crates/rslint_lexer/src/tests.rs

```diff
@@ -89,6 +89,19 @@ fn strip_shebang() {
     assert_eq!(lex.cur, 13);
 }
 
+#[test]
+fn shebang() {
+    assert_lex! {
+        "#! /bin/node",
+        SHEBANG:12
+    }
+    assert_lex! {
+        "#!/bin/node\n",
+        SHEBANG:11,
+        WHITESPACE:1
+    }
+}
+
```

Comment:
You should add a test for `\u{2028}` occuring at the end of the shebang, theres been some fuzz failures with it, it should not consume the first bytes of the char, it should leave the linebreak to the next line